src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -880,6 +880,11 @@ public void update() {
             });
         });
 
+        actions.put(Actions.OPEN_CONSOLE, (BaseAction) () -> {
+            JabRefDesktop.openConsole(((BasePanel) frame.tabbedPane.getComponentAt(frame.tabbedPane.getSelectedIndex()))
```

Comment:
Done. Yes, I only looked for appearances in JabRefFrame.java, I apologize for that.